Makefile

```diff
@@ -54,7 +54,7 @@ normal=\033[0m
 red=\033[0;31m
 
 prepare: check_venv
-	@$(pip) install toil==3.8.0 pytest==2.8.3
+	@$(pip) install toil==3.8.0 pytest==2.8.3 packaging
```

Comment:
This will actually break on a device that has a low version of pip. Drop this line.